modules/reitit-core/src/reitit/coercion.cljc

```diff
@@ -153,10 +171,15 @@
       (impl/fast-assoc response :body (coercer request response))
       response)))
 
-(defn request-coercers [coercion parameters opts]
-  (some->> (for [[k v] parameters, :when v]
-             [k (request-coercer coercion k v opts)])
-           (filter second) (seq) (into {})))
+(defn request-coercers
+  ([coercion parameters opts]
+   (some->> (for [[k v] parameters, :when v]
+              [k (request-coercer coercion k v opts)])
+            (filter second) (seq) (into {})))
+  ([coercion parameters request opts]
```

Comment:
Somewhat misleading naming here since `request` isn't an actual request, it's the `:request` key from the route data. `route-request`?

Reply:
I like `route-request` - changed.